src/Behat/Behat/Definition/Printer/ConsoleDefinitionPrinter.php

```diff
@@ -67,8 +67,11 @@ public function __construct(
      *
      * @param string $text
      */
-    final protected function write($text)
+    final protected function write($text, bool $addEmptyLine = false)
     {
+        if ($addEmptyLine) {
```

Comment:
We add the possibility to print an empty line before the text so that we can separate it from the previous output

Reply:
To me, `add` feels like it's going to append it - maybe `prependEmptyLine` or `lineBreakBefore` (which is really what it's doing - it doesn't necessarily add an *empty* line if something else has written characters without a linebreak...)?